packages/jellyfish-api-core/src/category/loan.ts

```diff
@@ -68,6 +68,16 @@ export class Loan {
     return await this.client.call('listloanschemes', [], 'bignumber')
   }
 
+  /**
+   * Get loan scheme.
+   *
+   * @param {string} id Unique identifier of the loan scheme
```

Comment:
I just confirmed with Diego. He said that there is no checking on whether the id is an empty string or more than characters in C++ side.